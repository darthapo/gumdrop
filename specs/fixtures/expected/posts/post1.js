(function() {
  var Test;

  Test = (function() {

    function Test() {
      this.img = $('<img src="../media/my-images.png">');
    }

    return Test;

  })();

}).call(this);
