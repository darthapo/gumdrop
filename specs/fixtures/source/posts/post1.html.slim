h1 My First Post

img src="/media/crap.png"

markdown:
  Go to [Hell](/about/hell)