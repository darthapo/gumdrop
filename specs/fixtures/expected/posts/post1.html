<h1>My First Post</h1><img src="../media/crap.png" /><p>Go to <a href="../about/hell">Hell</a></p>
